__tests__/app.test.js

```diff
@@ -113,3 +113,91 @@ describe("CORE: GET /api/articles", () => {
       });
   });
 });
+
+describe("CORE: GET /api/articles/:article_id/comments", () => {
+  test("GET: 200, should get all comments for an article", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((response) => {
+        const { comments } = response.body;
+        comments.forEach((comment) => {
+          expect(comment.article_id).toBe(1);
+        });
+      });
+  });
+  test("Comments should be served with the most recent comments first. ", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((response) => {
+        const { comments } = response.body;
+        expect(comments).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+  test("GET:404 responds with an appropriate status and error message when provided a valid but non existing article id", () => {
```

Comment:
We also want a test for the following:
Status 200, valid ID, but has no comments responds with an empty array of comments
Status 400, invalid ID, e.g. string of "not-an-id"